src/index.js

```diff
@@ -76,6 +76,28 @@ window.addEventListener("beforeinstallprompt", e => {
   }
 });
 
+async function fetchFeaturedRooms() {
+  // Fetch favorite rooms
+  const [favoriteRoomsResult, publicRoomsResult] = await Promise.all([
+    authChannel.signedIn
+      ? fetchReticulumAuthenticated(
+          `/api/v1/media/search?source=favorites&type=hubs&user=${store.credentialsAccountId}`
+        )
+      : Promise.resolve({ entries: [] }),
+    fetchReticulumAuthenticated("/api/v1/media/search?source=public_rooms")
+  ]);
+
+  if (favoriteRoomsResult && publicRoomsResult) {
+    publicRoomsResult.entries.push(...favoriteRoomsResult.entries);
+    const ids = publicRoomsResult.entries.map(h => h.id);
+    publicRoomsResult.entries = publicRoomsResult.entries
+      .filter((h, i) => ids.lastIndexOf(h.id) === i)
+      .sort((a, b) => b.participant_count - a.participant_count);
+    featuredRoomsResult = publicRoomsResult;
```

Comment:
this is a little weird, because the result we're passing to the UI is presumed to be a single API result, whereas this is not really an API result but a weird data structure that is two API results mashed together. I'd refactor the prop to just be the entries and update the value here to be the result of the `filter.sort`. if there are other things being accessed in `meta` for example in the subcomponent (I doubt it) then we should lift those to props as well.